two_factor_otp_auth/data/ir_actions_server_data.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+
+        <record id="ir_actions_server_user_enable_2fa" model="ir.actions.server">
+            <field name="name">Enable Two Factor Auth</field>
+            <field name="model_id" ref="model_res_users"/>
+            <field name="binding_model_id" ref="model_res_users"/>
+            <field name="binding_type">action</field>
+            <field name="state">code</field>
+            <field name="activity_user_type">specific</field>
+            <field name="code">action = records.action_enable_2f_auth()</field>
+        </record>
```

Comment:
@Werded , please, make sure that both this actions are visible on for user with Technical Settings.

Reply:
done